animegrep.sh

```diff
@@ -136,7 +136,7 @@ function cutvideo
 	
 	echo "$1" start time: "$2" end time: "$3" >> out/times.txt;
 
-	#ffmpeg -i "$1" -ss 0"$2" -to 0"$3" -async 1 -c:v libx264 -preset ultrafast out/clips/clip_"$x".mkv < /dev/null
+	ffmpeg -i "$1" -ss 0"$2" -to 0"$3" -async 1 -c:v libx264 -preset ultrafast out/clips/clip_"$x".mkv < /dev/null
```

Comment:
I might change it into something more balanced soon but you'll never know.

Reply:
yeah, if you're able to optimise it at all that would be great. I just used whatever I got working at the time there. I'm sure it can be a lot better